daemon/config.cfg

```diff
@@ -9,6 +9,9 @@ database=xmusic
 file=xmusic-daemon.log
 level=DEBUG
 
+[YOUTUBE]
+key=AIzaSyC02GiHCafXnwCjYN7e7djR9ogys0aEGlE
+
```

Comment:
If this key is for development or debugging, it's okay. But if it's for formal using, please remove this sensitive information out of the source control.